bottom-line/src/main/resources/partitionVariants.py

```diff
@@ -55,7 +56,6 @@
         .filter(df.beta.isNotNull() & ~isnan(df.beta)) \
         .filter(df.n.isNotNull()) \
         .select(
-            df.dataset,
```

Comment:
Why do we no longer need the dataset field?

Reply:
Ah this was a mistake I made initially. The partition bit below gets rid of the columns you are partitioning by. That's why they are added. And so since I'm no longer partitioning by dataset I have to remove it to make the ultimate table the same as it was before. Otherwise you get an extra (unnecessary) column and the trans-ethnic run flips out.